.env.example

```diff
@@ -7,3 +7,7 @@ LOG_LEVEL=debug
 
 # Go to https://smee.io/new set this to the URL that you are redirected to.
 WEBHOOK_PROXY_URL=
+
```

Comment:
I think we miss the `MNEMONIC` env

Reply:
We will use PRIVATE_KEY or something of the like, instead of mnemonic.